security_token.py

```diff
@@ -60,6 +60,11 @@ def approval_program():
             Or(new_admin_type == Bytes("contract admin"), new_admin_type == Bytes("transfer admin")),
             Txn.accounts.length() == Int(1)
         )),
+        # to avoid removing all contract admins, we do not allow contract admins to remove their own contract admin role
+        Assert(And(
+            new_admin_type == Bytes("contract admin"),
+            Neq(Txn.accounts[0], Txn.accounts[1])
```

Comment:
It seems like neither:
`Neq(Txn.accounts[0], Txn.accounts[1])`
or
``Neq(Txn.sender(), Txn.accounts[0])` check that the sending contract admin address cannot remove themself.